packages/@aws-cdk/aws-cloudwatch/lib/query.ts

```diff
@@ -0,0 +1,75 @@
+import * as cdk from '@aws-cdk/core';
+import { ConcreteWidget } from "./widget";
+
+/**
+ * Properties for a Query widget
+ */
+export interface QueryWidgetProps {
+  /**
+   * Title for the graph
```

Comment:
Everywhere in the docs where it says "graph" it should probably say "query widget" or just "widget".

Reply:
Right, missed that. I'll fix it in the next revision.